case-lib/config.sh

```diff
@@ -63,6 +63,11 @@ SUDO_LEVEL=${SUDO_LEVEL:-}
 #
 # NO_BT_MODE=true
 
+# Logger is not supported for firmware without ldc file. With NO_LOGGER_MODE
+# enabled, sof-logger will not be started.
+#
+# NO_LOGGER_MODE=true
+
```

Comment:
I am in favor of not using negative logic :). e.g LOGGER_MODE= true (we have logging) LOGGER_MODE=false (we don't have logging).

Reply:
Because the NO_XXX_MODE is minority, and XXX_MODE is majority. Normally, NO_XXX_MODE is set for special one or two devices, or otherwise we need to set XXX_MODE for all devices. 

Ex, in the early stage of new HW, graphics driver is not ready, HDMI pipelines are broken, so we only need set NO_HDMI_MODE for the new HW.

Reply:
> Because the NO_XXX_MODE is minority, and XXX_MODE is majority.

So make LOGGER_MODE=true and HDMI_MODE the default values and problem solved. There's never a good reason for negative options.